### PR TITLE
Remove extra space in IISWebAppDeploy help text

### PR DESCRIPTION
**Pull request by mderriey**
As mentioned in #203, feel free to contribute to that branch before merging if you feel like we could make things clearer.

### CODE REVIEW

File: Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppDeploy/task.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
       "required": false,
       "groupName": "deployment",
       "defaultValue": "",
-      "helpMarkDown": "Specify parameters here to override the ones in the Web Deploy zip file and in the Parameter file. To override more than one parameter use line separator, e.g., <br/> name=\"IIS Web Application Name\",value=\"Fabrikam\" <br/> name=\"ConnectionString\", value=\"Server=localhost;Database=Fabrikam;\""
+      "helpMarkDown": "Specify parameters here to override the ones in the Web Deploy zip file and in the Parameter file. To override more than one parameter use line separator, e.g., <br/> name=\"IIS Web Application Name\",value=\"Fabrikam\" <br/> name=\"ConnectionString\",value=\"Server=localhost;Database=Fabrikam;\""
```
Comment (mvvsubbu) [inline Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppDeploy/task.json]: can you update task patch version

**Comment by mderriey** [comment]
@mvvsubbu Do I also have to bump it in `vss-extension.json`?

**Comment by mvvsubbu** [comment]
No not required. That we will do while we are publishing.

**Comment by mvvsubbu** [comment]
This change I have merged as part of different PR. Thank you for your contribution

**Pull request closed without merging**